dynamics-nav/Setting-.NET-Framework-Types-to-Target-the-Microsoft-Dynamics-NAV-Windows-Client-or-Server.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Setting .NET Framework Types to Target the Microsoft Dynamics NAV Windows Client or Server"
+description: Setting .NET Framework Types to Target the Microsoft Dynamics NAV Windows Client or Server helps extend a navnow object by using .NET Framework interoperability.
```

Comment:
Maybe - "Describes how to set the type instance to target the nav_windows (client-side object) or nav_server (server-side object) in Microsoft Dynamics NAV."